docs/files.md

```diff
@@ -2,16 +2,25 @@
 
 This chapter explains in detail how you manage and review files in Reviewable.  The file matrix provides an overview of the files in the pull request, while the individual file areas show diffs and line-specific discussions.
 
+<a id="revision-cells"></a>
 ## Revision cells
 
 A file's evolution over the course of the pull request is summarized in both the file matrix and the file areas by a row of revision cells.  Each cell represents one revision, with the leftmost ⊥ cell being a virtual base revision that changes to match the right diff bound.  Colors and icons in the revision cells summarize what happened in the pull request at each revision, as well as the latest review marks.
 
-![reviewable file matrix status](images/filematrix_9.png)
+![reviewable file matrix status](images/filematrix_10.png)
 
 The brackets in a row are the current left and right diff bounds for the file. To adjust the diff bounds, click on one desired revision bound and drag to the other one.  You can also just click on a revision to adjust the nearest bound.
 
 ![reviewable file matrix diff bounds](images/filematrix_8.png)
 
+By design, Reviewable doesn't track files that haven't deviated from their base form -- including files that have been modified back to their original state.  However, under certain circumstances (e.g., [rebasing](#rebasing)), Reviewable will "force" a final revision when it otherwise wouldn't.  This is indicated by a reverted icon in a kept cell and is designed to prevent you from missing a critical diff.  Reviewable will also add special styling to cells when [rebased heuristics](#heuristics) are in play.
```

Comment:
This should all go into the rebasing section, since in fact that's the only circumstance where we'll need to force a revision.  Also, it's not entirely accurate:  we might need to force more than one revision (in case of multi-commit rebases), and the reverted icon isn't specific to forced revisions (any more).